api/src/db/db.js

```diff
@@ -30,10 +30,10 @@ module.exports = class Db {
       return {
         author: author.author,
         posts: posts
-        .filter(post => post.first_name === author.author)
-        .map(({title, link, content, date }) => {
-          return { title, link, content, date: date.toString() };
-        }),
+          .filter(post => post.first_name === author.author)
+          .map(({title, link, content, date}) => {
```

Comment:
Correct me if I'm wrong, here you are using all the data from the object right? 
would this make it more readable?
``` javascript
.map(post => ({...post, date: post.date.toString()})
```